cypress/e2e/pages/onDemandTV/index.cy.js

```diff
@@ -229,6 +235,29 @@ const liteTestSuites = testSuites.map(testSuite => {
   };
 });
 
+const atiAnalyticsliteTestSuites = atiAnalyticsTestSuites.map(testSuite => {
+  const excludedLiteTests = [
+    assertDropdownNavigationComponentView, // Dropdown navigation removed from all pages, as it requires JS
+    assertDropdownNavigationComponentClick, // Dropdown navigation removed from all pages, as it requires JS
+  ];
+
+  const liteSiteTests = testSuite.tests.filter(
+    test => !excludedLiteTests.includes(test),
+  );
+
+  // All lite enabled pages should have the Lite Site Summary component
+  liteSiteTests.push(assertLiteSiteSummaryComponentToMainSiteClick);
+
+  return {
+    ...testSuite,
+    path: getPathWithSuffix({ path: testSuite.path, suffix: '.lite' }),
+    applicationType: 'lite',
+    useReverb: false,
+    siteId: testSuite.service === 'magyarul' ? 134 : testSuite.siteId,
```

Comment:
I don't think we need this magyarul check here as there are no magyarul routes defined

```suggestion
    siteId: testSuite.siteId,
```